docs/cli.md

```diff
@@ -12,15 +12,15 @@ needs to be required in your support files and globally installed modules cannot
 ## Running specific features
 
 * Specify a [glob](https://github.com/isaacs/node-glob) pattern
-  * `$ cucumber-js features/**/*.feature`
+  * `$ npx cucumber-js features/**/*.feature`
```

Comment:
I don't think we need to run this here. Don't we expect users to always install it as dev dependency instead of as a temporary install? It needs to be require-able in step definitions anyway.